### ISSUE TITLE
Unable to test the SOAP Requests while integrating this solution with developer portal

### ISSUE THREAD

**Issue by rbapi** [nmasse-itix/soapui#1]
Received below error even after enabling CORS policy in APICAST.
Could not get a reponse from server. Check network connectivity and SSL/TLS certificates.

**Comment by nmasse-itix** [comment]
Yes, you are using a self-signed certificate for your APIcast gateway. So, you need to open the gateway URL and trust it, before trying to use the plugin. 

As a best practice, you could issue a Let's encrypt certificate for your APIcast gateway (LE certificates are trusted in most browser). You would get rid of this error message and the plugin would work all the time.
